src/types/Graph.ts

```diff
@@ -90,12 +90,11 @@ export interface CytoscapeMouseOutEvent extends CytoscapeBaseEvent {}
 
 // Graph Structures
 
-// Responses is a map of maps, all strings. e.g.:
-// { code0: {
-//     flags0: %traffic,
-//     flags1: %traffic
-//   }}
-export type Responses = object;
+export type ResponseDetail = {
+  flags: object; // string map flags->percentageOfTraffic
+  hosts: object; // string map host->percentageOfTraffic
+};
+export type Responses = object; // map responseCode:string -> ResponseDetail
```

Comment:
I would make this block a bit more specific and remove the `object` with index signatures i.e.
```typescript
// Graph Structures

type PercentageOfTrafficByFlag = {
  [flag: string]: string;
};

type PercentageOfTrafficByHost = {
  [host: string]: string;
};

export type ResponseDetail = {
  flags: PercentageOfTrafficByFlag;
  hosts: PercentageOfTrafficByHost;
};

export type Responses = {
  [responseCode: string]: ResponseDetail;
};

```

Reply:
+1, done.